tests/ember-data-initializers.js

```diff
@@ -1,8 +1,9 @@
 /* eslint no-extra-semi: "off" */
 
+import DS from 'ember-data';
+
 ;(function() {
   /* globals Ember */
-  /* globals DS */
```

Comment:
If we add `var DS = require('ember-data').default;` we can remove this globals declaration but we will need to add `/* globals require */`.